uint32  req_mode
---
bool    ok
uint32  current_mode
